x/group/server/msg_server.go

```diff
@@ -457,6 +461,33 @@ func (s serverImpl) CreateProposal(ctx types.Context, req *group.MsgCreatePropos
 		return nil, err
 	}
 
+	// Try to execute proposal immediately
+	if req.Exec == group.Exec_EXEC_TRY {
+		// Consider proposers as Yes votes
+		for i := range proposers {
+			ctx.GasMeter().ConsumeGas(gasCostPerIteration, "vote on proposal")
+			_, err = s.Vote(ctx, &group.MsgVoteRequest{
```

Comment:
The ref issue suggested we would just assume that all signers of the message would be considered yes votes. is it possible to just set the `YesCount` to the number of signers in the message and then exec? not entirely sure if that would work but at a glance it seems it could be faster than running `Vote` for each proposer

Reply:
I think the voters' weights can be more than 1 in some cases.